lib/grpc/GrpcMethods.ts

```diff
@@ -1,13 +1,13 @@
 import assert from 'assert';
-import { RpcComponents } from './RpcServer';
+import { GrpcComponents } from './GrpcServer';
 import Logger from '../Logger';
 import Pool from '../p2p/Pool';
 import OrderBook, { Order } from '../orderbook/OrderBook';
 import LndClient from '../lndclient/LndClient';
 import RaidenClient, { TokenSwapPayload } from '../raidenclient/RaidenClient';
 
 /** Class containing the available RPC methods for Exchange Union */
-class RpcMethods implements RpcComponents {
+class GrpcMethods implements GrpcComponents {
```

Comment:
Let's call this class GrpcService, because that's the terminology in the protobuf